pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -76,9 +77,9 @@ def is_empty(self):
 
 class ArrayQueue(Queue):
 
-    __slots__ = ['front']
+    __slots__ = ['front', 'rear', 'deque']
```

Comment:
Why do we need an extra, `rear`?

Reply:
The rear pointer is way to access the last element in the queue, As initial declaration of rear value will take up in the construction place and it will also help in deletion of queue and I have removed the default rear property.